client/homebrew/phbStyle/phb.style.less

```diff
@@ -468,3 +468,99 @@ body {
 		margin-bottom : 10px;
 	}
 }
+//*****************************
+// *      MAGIC ITEM CARD
+// *****************************/
+.phb .card {
+	width: var(--card-width);
+	height: var(--card-height);
+	-webkit-print-color-adjust:exact;
+	border: 2px solid black;
+	border-radius: 20px;
+	column-count: 1;
+	padding-left: 10px;
+	padding-right: 10px;
+	column-span: all;
+	margin: auto;
+	box-shadow: 0px 0px 15px #888 inset;
+}
+
+.phb .card.back {
+	width: calc(10px + (2 * var(--card-width)));
+	column-count: 2;
+	column-rule: 2px solid black;
+	column-gap: 25px;
+	padding-left: 0px;
+}
+
+.phb .card.back::before {
+	content: url(https://i.imgur.com/iNJH8j9.png) !important;
+}
+
+.phb .card h1 {
```

Comment:
Note: you could use [`parent selectors`](http://lesscss.org/features/#parent-selectors-feature) to structure this code in a better (I suppose) way

Reply:
This would also wrap the entire `card` style in a single set of braces, which should hopefully make it easier to understand what is and is not part of the style. I've rewritten the `phb.style.less` entries and am testing them now; if it works they way it should (in that it should be exactly the same) I'll push it into a commit.